tests/test_simple.py

```diff
@@ -21,6 +21,22 @@ def testEnvironment(self):
         self.assertIn(envVar, os.environ)
         self.assertTrue(os.path.exists(os.environ[envVar]), f"Check path {os.environ[envVar]}")
 
+    def testNoImplicitMultithreading(self):
+        """Test that the environment has turned off implicit
+        multithreading.
+        """
+        envVar = "OMP_NUM_THREADS"
+        self.assertIn(envVar, os.environ)
+        self.assertEqual(os.environ[envVar], "1")
+
+        try:
+            import numexpr
+        except ImportError:
+            numexpr = None
```

Comment:
```suggestion
        try:
            import numexpr
            self.assertEqual(numexpr.utils.get_num_threads(), 1)
        except ImportError:
            # It's not a problem if numexpr isn't available.
            pass
```

Reply:
I don't like putting extra code in the try block.

Reply:
Eh, the ImportError can only be raised by one thing in it... I don't care that much, but I think it's simpler this way.

Although: when would numexpr not be available in the environment?

Reply:
I don't know if sconsUtils is something that we ever may release separately and it doesn't really depend on numexpr, so I thought this was safer.  And I didn't want to do the import up top because of all the reasons that this is necessary in the first place.